mozregression/cli.py

```diff
@@ -392,7 +392,9 @@ def _convert_to_bisect_arg(self, value):
                                  % (new_value, value))
                 value = new_value
             except UnavailableRelease:
-                pass
+                self.logger.warning(
+                    "Target release version %s is not yet supported." % (value)
+                )
```

Comment:
Two things:

1. Could you change this from warning to error?
2. Could you add a "raise" statement here? This will make mozregression error out (which is what we want)